example-locale_api/README.md

```diff
@@ -0,0 +1,155 @@
+# Appwrite + VueJS =❤️
+
+This example is to showcase [Appwrite's Locale API](https://appwrite.io/docs/client/locale) with [Vue](https://vuejs.org/) by creating a simple web-app that displays your IP, Country,Continent,CountryCode,Continent and Currency.
+
+## Getting Started
+
+To get started quickly we will use vue cli to create the boilerplate that our code will be built on.
```

Comment:
Add some more punctuation in this line please

Reply:
Done.